lib/elixir/lib/supervisor/spec.ex

```diff
@@ -224,6 +224,17 @@ defmodule Supervisor.Spec do
 
   Check `Supervisor.Spec` module docs for more information on
   the options.
+
+  ## Examples
+
+  Here we start a `GenEvent` that is named `MyGenEvent`. We test that the
+  GenEvent is actually created with that name by calling a function that
+  `which_handlers`. As no handlers are added, it returns an empty list.
+
+      iex> genevent_worker = Supervisor.Spec.worker(GenEvent, [[name: MyGenEvent]])
```

Comment:
If GenEvent is being used only as an example, let's please use GenServer as it is much more common than GenEvent. :)